### PR TITLE
general improvements

### PR DESCRIPTION
**Pull request by solidus73**
aaaa

### CODE REVIEW

**Comment by solidus73** [comment]
pls respond

File: index.html [hunk 1]
```diff
@@ -1,14 +1,28 @@
-<!doctype html>
-<html>
 
+<!DOCTYPE html>
 <head>
-	<meta charset="utf-8"/>
-	<title>Resume of H. Silvestri</title>
-	<link rel="stylesheet" type="text/css" href="default-ss.css"> 
+<title>resumeme</title>
```
Comment (medsouz) [inline index.html:4]: Lacking indentation

File: index.html [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
+<style>
+@font-face {
+	font-family:Papyrus;
+	src: url("//soleedus.me/projects/resume/PAPYRUS.TTF") format('truetype');
+}
+h1 {color:#d23d4e;font-family: Papyrus, fantasy;font-size:600%;}
```
Comment (medsouz) [inline index.html:10]: Formatting errors, missing newlines

File: index.html [hunk 3]
```diff
@@ -1,14 +1,28 @@
...
+</style>
 </head>
-
 <body>
-	<h1>The Hunter Silvestri Index</h1>
-	<img src="huntercandid.jpg" alt="The titular Hunter Silvestri enjoying some sitting near childhood home.">
-	<h2><a href="https://huntersilvestri.github.io/bio.html">Biography</a>&emsp;&emsp;&emsp;<a href="https://huntersilvestri.github.io/resume.html">Resume</a></h2>
-</body>
\ No newline at end of file
+<img src="boi.gif">
```
Comment (medsouz) [inline index.html:22]: Missing image and improper indentation

File: index.html [hunk 4]
```diff
@@ -1,14 +1,28 @@
...
+<h1 style="-ms-transform: rotate(7deg); -webkit-transform: rotate(7deg); transform: rotate(7deg);"><br>web design is my passion</h1>
+<embed name="angst" loop="true" hidden="true" src="//soleedus.me/projects/resume/crawling.mp3"></embed>
+
+<!--stop looking at my code retard-->
```
Comment (medsouz) [inline index.html:26]: Triggered

File: index.html [hunk 5]
```diff
@@ -1,14 +1,28 @@
...
+<!DOCTYPE html>
 <head>
-	<meta charset="utf-8"/>
-	<title>Resume of H. Silvestri</title>
-	<link rel="stylesheet" type="text/css" href="default-ss.css"> 
+<title>resumeme</title>
+<style>
+@font-face {
+	font-family:Papyrus;
+	src: url("//soleedus.me/projects/resume/PAPYRUS.TTF") format('truetype');
```
Comment (medsouz) [inline index.html:8]: Do you have the rights to use this font? Please include a license.

**Review by medsouz — CHANGES_REQUESTED** [review]
Some suggestions to improve your code quality.

**Comment by medsouz** [comment]
Please squash these commits into one, three is unnecessary.

**Comment by Soleedus** [comment]
fuck matt he is very stink and Also not very Cool guy!!!!

**Pull request closed without merging**